source/Roadmap/Roadmap.md

```diff
@@ -0,0 +1,29 @@
+# Roadmap
+
+This page describes planned work for the ROS 2 Real-Time Working Group.
+
+
+| **Task**                                                     | **Size** | **Owner** | **Expected Completion** |
+| ------------------------------------------------------------ | -------- | --------- | ----------------------- |
+| Documentation: Complete Linux real-time guide                | Small    |           | 2nd quarter 2021        |
+| Documentation: Add introduction document                     | Small    |           | 2nd quarter 2021        |
+| Documentation: How to configure and run a ROS2 <br/> Real-Time application | Small    |           | 2nd quarter 2021        |
+| Documentation:  How to run run the build-farm <br/>real-time tests locally | Small    |           | 2nd quarter 2021        |
+| Documentation:  Build-farm setup description                 |          |           |                         |
+| Buidfarm: Implement infrastructure to run the <br/> ROS 2 real-time tests | Large    |           |                         |
+| Buidfarm: Provide hardware to deploy and run <br/> the ROS 2  real-time tests | Small    |           |                         |
+| Buidfarm: Improve existing tools to measure <br/> all ROS 2  different executors | Medium   |           |                         |
```

Comment:
Add item to define RT requirements (KPIs)

Reply:
Add item for "Carry out performance test measurements for ROS 2 stack" https://github.com/ros-realtime/community/issues/7

Reply:
Add Related issues, PRs column

Reply:
Add item for "Consolidate HW setting + OS setting"  https://github.com/ros-realtime/community/issues/13

Reply:
Add tutorial for https://github.com/ros2/examples/tree/master/rclcpp/executors/cbg_executor

Reply:
Separate tutorials in different sections like in micro-ros documentation: https://micro.ros.org//docs/tutorials/core/overview/